examples/logs/exporters/otlp_http.php

```diff
@@ -35,7 +35,7 @@
         (new LogRecordLimitsBuilder())->build()->getAttributeFactory()
     )
 );
-$logger = $loggerProvider->getLogger('demo', '1.0', 'http://schema.url', ['foo' => 'bar']);
+$logger = $loggerProvider->getLogger('demo', '1.0', 'https://opentelemetry.io/schemas/1.7.1', ['foo' => 'bar']);
```

Comment:
Will we update this link everytime version will change?

Reply:
yes, right

Reply:
That's a really good question. According to the docs, published schemas _should_ be there: https://opentelemetry.io/docs/specs/otel/schemas/#opentelemetry-schema
It doesn't stop the example from working, but I'll follow up with the other maintainers.